content/guides/ecommerce-connection/step2-understand-akeneo-pim.md

```diff
@@ -1,45 +1,54 @@
 # Understand Akeneo PIM
 
-## Introduction
+## Versions and editions
 
-### Where can I find the user documentation of Akeneo PIM?
+First, it is crucial for you to understand that Akeneo PIM exists in different flavours, depending on its version and edition.
 
-We could spend a lot of time explaining you each PIM's features but there are quite a lot of them... the easiest way is to consult our user documentation that you can find in our [Akeneo Help center](https://help.akeneo.com/pim/index.html).
+To know more about the different PIM versions, you can consult [this article](https://www.akeneo.com/blog/akeneo-introduces-a-simpler-product-release-cycle/) to understand our PIM release cycle.
 
-If you feel that a comprehensive training program would allow you to know everything about Akeneo PIM, we have a complete training program. Do not hesitate to register to one of our [training sessions](https://www.akeneo.com/training/).
+You can also consult our [release notes](https://www.akeneo.com/release-notes/) to understand the scope of each version.
 
-### What are Akeneo PIM versions?
+To finish, you'll find on our [helpdesk platform](http://helpdesk.akeneo.com/) what are our last supported PIM versions. If you don't have access to our Helpdesk platform, please [contact us](https://www.akeneo.com/contact/).
 
-First you can consult [this article](https://www.akeneo.com/blog/akeneo-introduces-a-simpler-product-release-cycle/) to understand our PIM release cycle.
+Then, in order to understand our different editions and their capabilities, you can consult our [dedicated page](https://www.akeneo.com/compare-editions/) on our website.
 
-Then, in order to understand our latest versions and their functional scope, you can consult our [release notes](https://www.akeneo.com/release-notes/) on our website.
+## Features and data model
```

Comment:
I don't understand this title and its unique subtitle below. There is only one subtitle which makes this title quite useless. What do you think?

Reply:
Or maybe it's "Acquire PIM knowledge" which is not at the right level and should also be a subtitle of this section?